Documentation~/customizing.md

```diff
@@ -1,83 +1,16 @@
-
-In addition, you can customize several things about any of the toolbars that ProBuilder provides:
-
-* You can [dock the toolbar or let it float](#dock) over the Unity window.
-* You can [resize the toolbar](#resize) both horizontally and vertically. 
-
-You can also choose whether to display [text or icons](toolbar.md#buttonmode) on the ProBuilder toolbar.
-
-<a name="buttons"></a>
-
-### Text vs Icon mode
-
-To define whether ProBuilder displays [text or icons](toolbar.md#buttonmode) for each tool:
-
-1. Right-click on any empty space in the toolbar. 
-
-	![Use Text Mode or Use Icon Mode](images/cus_switchmodes.png)
-
-2. Choose either **Use Text Mode** or **Use Icon Mode** from the context menu.
-
-> **Note:** You can also use the __Use Icon GUI__ option in the [ProBuilder Preferences](preferences.md#icongui) window.
-
-
-
-<a name="dock"></a>
-
-### Floating vs Dockable
-
-To change whether the toolbar [docks or floats](https://docs.unity3d.com/Manual/CustomizingYourWorkspace.html) in the Scene view:
-
-1. Right-click on any empty space in the toolbar.
-
-  ![Use Floating or Dockable Window](images/cus_switchwindows.png)
-
-2. Choose either **Open As Floating Window** or **Open As Dockable Window** from the **Window** submenu of the context menu.
-
-
-
-<a name="resize"></a>
-
-### Resizing the toolbar
-
-To resize the ProBuilder toolbar, click and drag the corners and sides of the toolbar until it is at the right size and shape. 
-
-ProBuilder re-orders the icon or text contents to best fit the window size.
-
-![Text vs Icon modes](images/Toolbar_ResizingExample.png)
+# Customize ProBuilder
+
+You can customize ProBuilder and set preferences in the [ProBuilder Preferences](preferences.md) window. You can set the following preferences: 
+
+| **Preference type** | **Description** |
+| --- | --- |
+| [Dimensions Overlay](preferences.md#dimover) | Customize which elements the Dimensions overlay uses. |
+| [Experimental](preferences.md#experimental) | Toggle ProBuilder beta features. |
+| [General](preferences.md#general) | A variety of settings, many of which control what information appears in the **Scene** view. For example, you can see the dimensions of your mesh, how many vertices, edges, and faces are in the scene and which ones are selected. |
+| [Graphics](preferences.md#graphics) | Control the colors the Unity Editor uses when you edit. You can use the Unity color scheme, or disable the [Use Unity Colors](preferences.md#unitycolors) preference to set your own colors for the ProBuilder geometry elements. |
```

Comment:
Doesn't look like `unitycolors` setting exists. I can't find it. Must be an artifact. Also, it looks like Graphics does a bit more than set the colors too. 
```suggestion
| [Graphics](preferences.md#graphics) | Control the colors the Unity Editor uses when you edit. You can use the Unity color scheme, or disable the Use Unity Colors preference to set your own colors for the ProBuilder geometry elements. |
```